lib/client/index.js

```diff
@@ -0,0 +1,11 @@
+var mediatorSubscribers = require('./mediator-subscribers');
+
+/**
+ * Initialisation of the workorder core module.
+ * @param {Mediator} mediator
+ */
+module.exports = function(mediator) {
+
+  //Initialising the subscribers to topics that the module is interested in.
+  return mediatorSubscribers.init(mediator);
```

Comment:
Most of the modules using pattern where client is passed as second parameter. In this particular case client is static, but I think we should use the same patterns across the board to simplify refactoring.